app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@ class UsersController < ApplicationController
   before_action :authenticate_user!, only: %i[give_kudo]
 
   def index
-    @users = User.paginate(page: params[:page], per_page: 15)
+    @users = User.all.sort_by { |user| [user.received_kudos.count] }.reverse.paginate(page: params[:page], per_page: 15)
```

Comment:
What about extracting `User.all.sort_by { |user| [user.received_kudos.count] }.reverse` to e.g. `sorted_users` variable? I think it would make this method more readable (and line a little bit shorter)